docs/source/content.rst

```diff
@@ -176,7 +176,13 @@ Updating a Resource with PATCH
 ------------------------------
 
 To update an existing resource we send a PATCH request to the server.
-PATCH allows to provide just a subset of the resource (the values you actually want to change):
+PATCH allows to provide just a subset of the resource
+(the values you actually want to change).
+
+If you send the value `null` for a field, the field's content will be
```

Comment:
In reStructuredText these need to be double backticks (``) for code literals, single backticks are just emphasis 😉